src/adapter/src/metrics.rs

```diff
@@ -206,7 +207,12 @@ impl Metrics {
                 help: "The time it takes to run `ensure_transactions` when processing pgwire messages.",
                 var_labels: ["message_type"],
                 buckets: histogram_seconds_buckets(0.001, 512.0),
-            ))
+            )),
+            tokio_canary_delay_seconds: registry.register(metric!(
```

Comment:
should we be optimistic and call this `mz_tokio_canary_delay_ms`, and use ms as the base unit? 😅

Reply:
Yep, I'll do that! :sweat_smile: